.github/workflows/release.yml

```diff
@@ -0,0 +1,56 @@
+# Release and deploy
+name: release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+
+      - name: Build
+        run: |
+          yarn install
+          yarn run build
+
+      - name: zip hypercrx.zip
+        run: zip release/hypercrx.zip -r build/*
+
+      - name: Release pre-Release
+        uses: "marvinpinto/action-automatic-releases@latest"
+        with:
+          repo_token: "${{ secrets.GITHUB_TOKEN }}"
+          automatic_release_tag: "latest"
+          prerelease: true
+          title: "Development Build"
+          files: |
+            release/hypercrx.crx
+            release/hypercrx.zip
+
+      - name: Release
+        if: startsWith(github.ref, 'refs/tags/')
```

Comment:
Hey @wxharry, would you mind adding some comment on this line so those not familiar with GitHub Actions can know this step won't happen except for PRs with tags.

And I wonder: if PRs which have tags but not tagged with the newest commit will trigger this step?